### DISCUSSION TITLE
404: not found when production deployment: ID sin1::2wvq9-1695723637180-40dbf005404b

### DISCUSSION THREAD

**Discussion by justnelling** [vercel/community#4246] [Help] [answered]
### Summary

Deployment with my site worked previously, but after changing text in one of the pages it now returns 404: NOT_FOUND again.

I have already included a vercel.json file in the root folder, with `{
  "rewrites": [{ "source": "/(.*)", "destination": "/index.html" }]
}`

Am not sure why htis is happening.

### Example

https://threejs-custom-shirt-designer-alpha.vercel.app/

### Steps to Reproduce

404: NOT_FOUND
Code: NOT_FOUND
ID: sin1::5mf6q-1695723458742-b899df781ecb

**Comment by kienvuforwork** [comment]
I got the same problem. Did you find the solution?

**Comment by MrKeem** [comment]
I  see your web could deploy, How you solve this problem

**Reply by justnelling** [reply, marked as answer]
hey ok so I moved the vercel.json to my frontend "./client" directory and then it worked. It didn't work when I was leaving it in the parent directory that held both my "./client" and "./server" side
